rbfe_tutorial/cli_tutorial.md

```diff
@@ -215,37 +215,34 @@ To gather all the $\Delta G$ estimates into a single file, use the `openfe
 gather` command from within the working directory used above:
 
 ```bash
-openfe gather ./results/ -o final_results.tsv
+openfe gather results/ --report dg -o final_results.tsv
 ```
 
-This will write out a tab-separated table of results, including both the
-$\Delta G$ for each leg and the $\Delta\Delta G$ computed from pairs of legs.
-The first column is a description of the data, e.g., `DGcomplex(ligandB,
-ligandA)` for the $\Delta G$ of the transformation of ligand
-A into ligand B complexed to a protein, or `DDGbind(ligandB, ligandA)` for the binding
-$\Delta\Delta G$ going from ligand A to ligand B. The second column tells the type of
-the result, either `RBFE` for a relative result or `solvent`/`complex` for an
-individual leg. The next two columns are the labels of the ligands, and then
-the computed result and its uncertainty.
+This will write out a tab-separated table of results where the results 
+reported are controlled by the `--report` option:
+
+  * `dg` (default) reports the ligand and the results are the maximum
+    likelihood estimate of its absolute free, and the uncertainty in
+    that.
```

Comment:
😅 maybe "associated uncertainty from DDG replica averages and standard deviations"?

Reply:
Yah I was going to say "that measurement" or something but I like what you have. I'll also update the CLI help message as well